Resources/doc/cookbook/recipe_dynamic_form_modification.rst

```diff
@@ -45,5 +45,8 @@ Then, you should be able to dynamically add needed fields to the form:
             if ($this->isCurrentRoute('create')) {
                 $formMapper->add('name', 'text');
             }
+
+            //$this->isCurrentRoute('create'); // is create page?
```

Comment:
Let's do this. Documentation improve can be done on another PR if wanted.

cc @Rivsen